django_cas_ng/backends.py

```diff
@@ -88,6 +88,13 @@ def authenticate(self, request: HttpRequest, ticket: str, service: str) -> Optio
         if pgtiou and settings.CAS_PROXY_CALLBACK and request:
             request.session['pgtiou'] = pgtiou
 
+        # Map CAS affiliations to Django groups
+        if settings.CAS_MAP_AFFILIATIONS and user and attributes:
+            affils = attributes.get('affiliation', [])
+            for affil in affils:
+                g = Group.objects.get_or_create(affil)
```

Comment:
Minor: If server response an empty string "" as affiliation, we should not create group name with empty string?

Reply:
Yes - good point. I've added a check for that here.